pco2_Graffiti_Kanekal_Braun.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep  8 13:27:43 2020
+
+@author: mackenziebraun
+"""
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+style=('courier', 40, 'bold')
+color('black')
+up()
+speed(0)
+goto(110,-110)
+write('pajAMAZ ON!', font=style, align='left')
+color('orange')
+
```

Comment:
First thing first, please comment before a paragraph. I don't know what this paragraph makes exactly, and for me as a third person it's important to do.